woocommerce/woocommerce-android
# Analytics bundle UI

Closes: #10967
### Why
This PR adds the basic changes needed to add and display the bundle stats card to the Analytics Hub screen.

### Description
This PR adds a new `bundlesState` field in the updated stats use case, which will notify the UI of the bundle's stats changes. The values retrieved by this use case are not connected to the API and are only for testing propose. For displaying the bundle stats card, we are reusing the same design and component as for all the analytics UI list cards, as we agreed on p1711470851988749-slack-C02KUCFCSFP.

### Testing instructions
1. Open the app
2. Tap on See all store analytics
3. Check that the bundle stats card is displayed

### Images/gif
<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/fd1ca379-689e-441d-bf6e-9e7e323bff39" />
